crazyflie/scripts/crazyflie_server.py

```diff
@@ -243,6 +246,11 @@ def __init__(self):
                 "/cmd_hover", partial(self._cmd_hover_changed, uri=uri), 10
             )
 
+            self.create_subscription(
+                NamedPoseArray, "/poses", 
+                self._poses_changed, qos_profile_sensor_data
```

Comment:
I just tested it! but the server does not receive the poses so there is probably QOS incompatibility with the topic I think. I will need to study the ROS py api for QOS a bit to see what it is missing: https://docs.ros2.org/foxy/api/rclpy/api/qos.html

Reply:
Setting the duration is important, that it what makes it incompatible.

Reply:
I figured out the right right way :) it seems to work now